common/src/main/java/io/homeassistant/companion/android/common/data/integration/Entity.kt

```diff
@@ -41,6 +41,7 @@ object EntityExt {
     val LIGHT_MODE_NO_BRIGHTNESS_SUPPORT = listOf("unknown", "onoff")
     const val LIGHT_SUPPORT_BRIGHTNESS_DEPR = 1
     const val LIGHT_SUPPORT_COLOR_TEMP_DEPR = 2
+    const val ALARM_CONTROL_PANEL_SUPPORT_ARMED_AWAY = 2
```

Comment:
The HA constant is for the feature/call, not the state: `ALARM_CONTROL_PANEL_SUPPORT_ARM_AWAY`